### ISSUE TITLE
Getting "FixedDataTable expected an ES6 compatible `Object.assign` polyfill" error

### ISSUE THREAD

**Issue by songz** [facebook/fixed-data-table#11]
Browser console error gives: 
FixedDataTable expected an ES6 compatible `Object.assign` polyfill.

**Comment by michaelcarter-wf** [comment]
Try using [es6-shim](https://github.com/paulmillr/es6-shim/). Solved the issue for me.

**Comment by pieterv** [comment]
@songz like @michaelcarter-wf said you will need an es6 Object.assign shim for the code to work. es6-shim will work, or a light weight option like https://github.com/sindresorhus/object-assign then do `Object.assign = require('object-assign');` would also work.

Please let us know if you can think of a more descriptive error message that would help you solve this issue faster.

**Comment by nanodocumet** [comment]
How can this be fixed when using `webpack`?

I have installed this package using `npm install --save fixed-data-table`, then I have the code below, but I am still getting the error. I am not sure if it is something that I need to fix in my webpack config or anywhere else. Any help will be appreciated!

Thanks.

```javascript
var React = require('react');
var FixedDataTable = require('fixed-data-table');

var Table = FixedDataTable.Table;
var Column = FixedDataTable.Column;

// Table data as a list of array.
var rows = [
  ['a1', 'b1', 'c1'],
  ['a2', 'b3', 'c2'],
  ['a3', 'b3', 'c3'],
  //..... /// and more
];

function rowGetter(rowIndex) {
  return rows[rowIndex];
}

var MyComponent = React.createClass({
  mixins: [FluxMixin, State],

  render: function() {
    return (<Table
      rowHeight={50}
      rowGetter={rowGetter}
      rowsCount={rows.length}
      width={5000}
      height={5000}
      headerHeight={50}>
      <Column
        label="Col 1"
        width={3000}
        dataKey={0}
      />
      <Column
        label="Col 2"
        width={2000}
        dataKey={1}
      />
    </Table>);
  }
});

module.exports = MyComponent;
```

**Comment by pieterv** [comment]
Just a note to say with #102 the `Object.assign` polyfill will no longer be needed :)

**Comment by franleplant** [comment]
This does not work with es6 imports somehow :(

**Comment by franleplant** [comment]
It seems like babel (which Im using to to transpile es6 to es5) is moving all module declarations to the top and that is why I cannot do the `Object.assign = Object.assign || require('object-assign');`

What I ended doing is to paste verbatim the ponyfill in a script in my `index.html` and it works. But it's a horrible hack and I will love to see #102 merged!

**Comment by pieterv** [comment]
Hey sorry this is such a pain, #102 will be out soon :P If you want in the mean time you can run the objectAssignTransformer yourself which should fix the issue for you, so grab the `objectAssignTransformer.js` file from the PR then add `"plugins": ["./path/to/my/stuff/objectAssignTransformer.js"],` to your babel config and that should fix it.

**Comment by franleplant** [comment]
Hi @pieterv no need to apologize :)
Last night I was kind of frustrated, nevertheless now it works and with #102 on the way there is nothing less to be said but what has already been said.

Thanks *a lot* for all your support.

Fran

**Comment by cendy172** [comment]
Hi there,
After I added Object.assign = Object.assign || require('object-assign'); before require fixed-data-table. the error in browser console is still:  "FixedDataTable expected an ES6 compatible `Object.assign` polyfill" and plus another error: Uncaught TypeError: Object.assign is not a function

Anyone can help me here?

**Comment by pieterv** [comment]
Hey @cendy172, Update to the new version we just released (0.2.0) and this issue should be fixed, we don't need the polyfill anymore :) Let me know if not.
